src/alchemlyb/estimators/ti_.py

```diff
@@ -30,14 +30,22 @@ class TI(BaseEstimator, _EstimatorMixOut):
     dhdl : DataFrame
         The estimated dhdl of each state.
 
+    _gq_delta_f_ : DataFrame, optional
+        The estimated dimensionless cumulative sum of free energy up to each state 
+        with gaussian quadrature.
+
+    _gq_d_delta_f_ : DataFrame, optional
+        The estimated statistical uncertainty (one standard deviation) in 
+        the cumulative sum of free energy up to each state with gaussian quadrature
```

Comment:
Will users want to access these data structures? If yes, no underscore and document them as you do.

If they are private attributes then they don't need to be documented.